homeassistant/components/conversation/default_agent.py

```diff
@@ -492,35 +492,7 @@ def _get_or_load_intents(
         # en-US, en_US, en, ...
         language_variations = list(_get_language_variations(language))
 
-        # Check if any new components have been loaded
         intents_changed = False
```

Comment:
```suggestion
        intents_changed = False
```

Reply:
```suggestion
        intents_changed = False
```

Reply:
```suggestion
        intents_changed = False
```

Reply:
@sedmasport27 did you mean to suggest changes? Github is saying the suggestions are invalid because no changes were made to the code.